frontend/packages/kubevirt-plugin/src/components/vms/virtualization.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import { withStartGuide } from '@console/internal/components/start-guide';
+import { HorizontalNav } from '@console/internal/components/utils';
+import { VirtualMachinesPage } from './vm';
+import { VirtualMachineTemplatesPage } from '../vm-templates/vm-template';
+import { VirtualMachineModel } from '../../models';
+import { FLAGS } from '@console/shared';
+import { connectToFlags, FlagsObject } from '@console/internal/reducers/features';
+import { compose } from 'redux';
```

Comment:
nitpick: I think the redux import should come before the console imports and the two console imports should be moved to be with the other console imports above.

Reply:
Thanks ! fixed